### ISSUE TITLE
Add new base network to spacer.

### ISSUE THREAD

**Issue by beijbom** [beijbom/coralnet#237]
Implement the pytorch extractor in 
https://github.com/beijbom/pyspacer/blob/master/spacer/extract_features.py

**Comment by beijbom** [comment]
Once https://github.com/beijbom/coralnet/issues/284 is done, which is more or less is, this should be much more straight-forward. All we need to do is to provide an implementation of 
```
class FeatureExtractor(abc.ABC):
```
in
https://github.com/beijbom/pyspacer/blob/master/spacer/extract_features.py.

**Comment by beijbom** [comment]
hey @qiminchen : would you mind taking a crack at this now? I understand you don't have the final weights and such, but for now you can just use whatever works best right now. This would help make sure we catch any integration problems sooner rather than later. You  can then update with the production model later. 

I should be straight-forward:
- Implement a FeatureExtractor in https://github.com/beijbom/pyspacer/blob/master/spacer/extract_features.py
- Put whatever pytorch utils you need in a torch_utils.py
- You can send me the weights and I'll upload them to the `spacer-tools` bucket. Then you can just call `download_model` (in storagey.py).
- Write tests of all utils and methods.

**Comment by qiminchen** [comment]
Sure, I will work on it right after evaluating the available test sources I have now, the results table in the research plan would be more decent by tomorrow and next Wednesday. Btw I have a final and presentation next week so I probably won't make huge progress on this issue this week but will catch up as soon as possible.

**Comment by qiminchen** [comment]
Hi Oscar,

Can you please upload the efficientnetb0 weight to the spacer-tool bucket?
The weight is attached below. I am working on the FeatureExtractor. Thanks.

cheers,
Qimin

 efficientnetb0_5eps_best.pt
<https://drive.google.com/file/d/1T5S2UA5x5N4PuKrAuxOGRxHkEwz3h9X9/view?usp=drive_web>

**Comment by beijbom** [comment]
Cool. I just uploaded it. LMK if it works and that the permissions are
right.

**Comment by qiminchen** [comment]
@beijbom Hi Oscar, I started the new dataset uploading yesterday and it should take a few days so I start working on this PR. I can download the weight and the permissions are right.

**Comment by beijbom** [comment]
Cool. Sounds good.

**Comment by beijbom** [comment]
@qiminchen : I'm considering this ticket done now with https://github.com/beijbom/pyspacer/pull/15 merged. Can you create a new ticket to integrate the final production network? Should be quick.
